src/hsmm_mvpy/models.py

```diff
@@ -188,8 +188,7 @@ def cross_correlation(self,data):
         events = np.zeros(data.shape)
         for trial in range(self.n_trials):#avoids confusion of gains between trials
             for dim in np.arange(self.n_dims):
-                dat = data[self.starts[trial]:self.ends[trial]+1, dim]
-                events[self.starts[trial]:self.ends[trial]+1,dim] = correlate(dat, self.template, mode='same', method='direct')
+                events[self.starts[trial]:self.ends[trial]+1,dim] = correlate(data[self.starts[trial]:self.ends[trial]+1, dim], self.template, mode='same', method='direct')
```

Comment:
Isn't that a residual from PR #118 ?

Reply:
Yes but it doesn't change anything and is slightly more efficient.

Reply:
Ah yes I didn't pay attention to the first removed line